contrib/babelfishpg_unit/test_money.c

```diff
@@ -0,0 +1,298 @@
+#include "babelfishpg_unit.h"
+#include "../babelfishpg_money/fixeddecimal.c"
+
+/*
+ * These are the test functions which returns a TestResult structure containing the result of the test.
+ */
+
```

Comment:
We should have comment stating what it is testing. Similarly for other tests as well.

Reply:
Okay I added comments for each test function about the functionality of the function it is testing